.github/workflows/dependency-review.yml

```diff
@@ -0,0 +1,27 @@
+# Avoids introducing vulnerable dependencies to the main branch.
+
+name: Dependency review
+on:
+  pull_request:
+
+jobs:
+  dependency-review:
+    runs-on: ubuntu-latest
+
+    permissions:
+      # To write dependency submission
+      contents: write
+      # To write comment on PR
+      pull-requests: write
+
+    steps:
+      - uses: actions/checkout@v5
```

Comment:
Could we use the commit SHAs for the versions of all the actions in this file, as per the dependency submission workflow?

Reply:
Done in 5bdf367252f074fbe984cc2d06083b2810857056